src/pages/Home.vue

```diff
@@ -22,10 +22,7 @@
         </p>
         <div class="wrap-cta">
           <p>Want to know more about us?</p>
-          <a
-            href="https://localp2p.money/local-litepaper.pdf"
-            target="_blank"
-          >
+          <a href="https://localp2p.money/local-litepaper.pdf" target="_blank">
```

Comment:
We should change this url from https://localp2p.money/local-litepaper.pdf to https://localmoney.io/local-litepaper.pdf

Reply:
Done.